release-notes/elastic-cloud-serverless/index.md

```diff
@@ -1620,6 +1734,13 @@ Review the changes, fixes, and more to {{serverless-full}}.
 * Introduces GA support for the [`semantic_text`](elasticsearch://reference/elasticsearch/mapping-reference/semantic-text.md) field type on {{serverless-full}}
 * Adds the ability for users to [customize prebuilt rules](https://github.com/elastic/kibana/issues/174168). Users can modify most rule parameters, export and import prebuilt rules — including customized ones — and upgrade prebuilt rules while retaining customization settings [#212761]({{kib-pull}}212761)
 
+* Propagates product use case http header to `EIS` [#124025](https://github.com/elastic/elasticsearch/pull/124025)
+* Integrates with DeepSeek API [#122218](https://github.com/elastic/elasticsearch/pull/122218)
+* Adds `max.chunks` to `EmbeddingRequestChunker` to prevent `OOM` [#123150](https://github.com/elastic/elasticsearch/pull/123150)
```

Comment:
```suggestion
* Adds `max.chunks` to `EmbeddingRequestChunker` to prevent out of memory error [#123150](https://github.com/elastic/elasticsearch/pull/123150)
```